File-Formats/PDF/Convert-HTML-To-PDF/troubleshooting.md

```diff
@@ -8,50 +8,33 @@ documentation: UG
 
 # Troubleshooting and FAQ
 
-## BlinkBinaries are missing
+## Blink files are missing
 
 <table>
 <th style="font-size:14px" width="100px">Exception</th>
-<th style="font-size:14px">BlinkBinaries are missing</th>
+<th style="font-size:14px">Blink files are missing</th>
 <tr>
 <th style="font-size:14px" width="100px">Reason
 </th>
-<td>When BlinkBinaries are not available in the <i>BlinkPath </i> mentioned location.
+<td>The exception may occurs if the '<i>runtimes</i>' folder doesn't get copied properly from the NuGet folder.
 </td>
 </tr>
 <tr>
 <th style="font-size:14px" width="100px">Solution</th>
 <td>
-Set the path of the BlinkBinaries folder to the BlinkPath property of BlinkConverterSettings.
+Make sure that after installing the NuGet package and building, the 'runtimes' folder will be copied automatically.
```

Comment:
Ensure that upon NuGet package installation and building, the 'runtimes' folder is automatically copied.